server.js

```diff
@@ -0,0 +1,203 @@
+});
+
+awsConfig.config.update({
+  accessKeyId: nconf.get('AWS_ACCESS_KEY'),
+  secretAccessKey: nconf.get('AWS_SECRET_KEY'),
+});
+
+_config({
+  verbose: false,
+  // This is the path to the directory containing iap-sanbox/iap-live files
+  googlePublicKeyPath: nconf.get('IAP_GOOGLE_KEYDIR'),
+  googleAccToken: nconf.get('PLAY_API_ACCESS_TOKEN'),
+  googleRefToken: nconf.get('PLAY_API_REFRESH_TOKEN'),
+  googleClientID: nconf.get('PLAY_API_CLIENT_ID'),
+  googleClientSecret: nconf.get('PLAY_API_CLIENT_SECRET'),
+  applePassword: nconf.get('ITUNES_SHARED_SECRET'),
+});
+
+const redisOpts = {
+  port: nconf.get('REDIS_PORT'),
+  host: nconf.get('REDIS_HOST'),
+};
+
+if (nconf.get('NODE_ENV') === 'production') {
+  const redisURL = parse(nconf.get('REDIS_URL'));
+  [, redisOpts.auth] = redisURL.auth.split(':');
+  redisOpts.host = redisURL.hostname;
+  redisOpts.port = redisURL.port;
+}
+
+const queueOpts = {
+  defaultJobOptions: {
+    attempts: 5,
+    backoff: {
+      type: 'exponential',
+      delay: 30 * 60 * 1000,
+    },
+  },
+};
+
+const comQueue = Queue('Communication', redisOpts, queueOpts);
+comQueue.process('email', 30, email);
+comQueue.process('sendBatchEmails', sendBatchEmails(comQueue, db, BASE_URL));
+comQueue.process('sendOnboardingEmails', sendOnboardingEmails(comQueue, db, BASE_URL));
+comQueue.process('sendSpecialPushNotifications', sendSpecialPushNotifications(comQueue, db));
+const paymentQueue = Queue('Payments', redisOpts, queueOpts);
+paymentQueue.process('amazonPayments', amazonPayments(paymentQueue, db));
+paymentQueue.process('googlePayments', googlePayments(paymentQueue, db));
+paymentQueue.process('applePayments', applePayments(paymentQueue, db));
+paymentQueue.process('amazonGroupPlanPayments', amazonGroupPlanPayments(paymentQueue, db));
+const remindersQueue = Queue('Reminders', redisOpts, queueOpts);
+remindersQueue.process('applePaymentsReminders', appleSubscriptionReminders(remindersQueue, db, BASE_URL));
+remindersQueue.process('googlePaymentsReminders', googleSubscriptionReminders(remindersQueue, db, BASE_URL));
+remindersQueue.process('amazonPaymentsReminders', amazonPaymentsReminders(remindersQueue, db, BASE_URL));
+remindersQueue.process('stripeReminders', stripeReminders(remindersQueue, db, BASE_URL));
+remindersQueue.process('paypalReminders', paypalReminders(remindersQueue, db, BASE_URL));
+remindersQueue.process('stripeGroupsReminders', stripeGroupsReminders(remindersQueue, db, BASE_URL));
+remindersQueue.process('amazonGroupsReminders', amazonGroupsReminders(remindersQueue, db, BASE_URL));
+remindersQueue.process('expirationReminders', expirationReminders(remindersQueue, db, BASE_URL));
+
+const queues = [
+  comQueue,
+  paymentQueue,
+  remindersQueue,
+];
+
+queues.forEach(queue => {
+  queue.on('completed', job => {
+    job.remove();
+  });
+  queue.on('failed', (job, error) => {
+    console.log(error);
+    /* const args = Array.prototype.slice.call(arguments);
```

Comment:
More commented out code? I understand on the recap emails, it's a good reminder that such a thing exists and could be enabled again, but what about this?